src/MigrationTools.Host/MigrationTools.Host.csproj

```diff
@@ -1,16 +1,16 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-	 <TargetFrameworks>netstandard2.0;net8.0</TargetFrameworks>
```

Comment:
The `netstandard2.0` projects are shared between the `net8.0` and `net472`... there is probably a better way to use `<TargetFrameworks>net472;net8.0</TargetFrameworks>` instead